### ISSUE TITLE
For contacts popup only display fields that are present for that contact

### ISSUE THREAD

**Issue by matthewberryman** [MSFREACH/msf-reach#187]
Already partly done but will need to keep track of this as #176 is done.

**Comment by matthewberryman** [comment]
@LucieGueuning on the contacts tab of the fields sheet can you indicate what items you want in the map popups vs the contact modal (that pops up when you click on the eye icon on events page). Or would you like the map popup to be the same as the contact modal?

**Comment by LucieGueuning** [comment]
working on that

**Comment by LucieGueuning** [comment]
1. Map popup
--
Last Name + First Name
phone number
email address
type of contact
specialty

2. Contact modal
--
All others fields stipulated into spreadsheet

**Comment by LucieGueuning** [comment]
@matthewberryman is that now the case on prod env?

**Comment by matthewberryman** [comment]
Yes the popups are correct and in prod env: note as per request all fields are listed, but if someone hasn't entered anything for non-mandatory fields on contact cards, then it lists nothing for it, e.g.:
<img width="139" alt="screenshot 2018-02-08 11 14 15" src="https://user-images.githubusercontent.com/2288238/35948574-45948ff2-0cc1-11e8-8d7e-955e4f671d2c.png">

A lot of the work on the modal is done and in prod, although there are some recent changes (e.g. fax and web) that are missing and I will fix those soon.

**Comment by matthewberryman** [comment]
Also the modal on events page was updated but not on landing page.

**Comment by matthewberryman** [comment]
@LucieGueuning Note that the default in the contact modal is to display all the section headers (social media, contact details, etc.) except that "MSF details" /  "employment details" depends on type being MSF staff or not, however the individual fields (e.g. whatsapp / twitter /etc.) are only shown if there is something for that. So a bit different to the popup. I'm wary of making it display all, as otherwise it can look cluttered and also take up too much space esp. small screens. In this recent work I've also made the emails as mailto: hyperlinks, and turned the web address into a hyperlink (adding `http://` if left off on contact input).

**Comment by LucieGueuning** [comment]
all good on that one?
